torch/package/package_importer.py

```diff
@@ -44,7 +44,7 @@ class PackageImporter(Importer):
     """The dictionary of already loaded modules from this package, equivalent to `sys.modules` but
     local to this importer.
     """
-    modules: Dict[str, Optional[types.ModuleType]]
+    modules: Dict[str, types.ModuleType]
```

Comment:
Isn't this inherited from `Importer`? Or do ABCs work differently?

Reply:
yeah it is, I guess it's redundant